components/kyma-environment-broker/common/runtime/model.go

```diff
@@ -59,6 +59,17 @@ const (
 	RegionParam          = "region"
 	ShootParam           = "shoot"
 	PlanParam            = "plan"
+	StateParam           = "state"
+)
+
+const (
+	RuntimeSucceeded      = "succeeded"
```

Comment:
Exported variables name should not start with the package name - consider renaming those, for example to `StateSucceeded`. You can also create a dedicated type `RuntimeState` like you did in `dbmodel/instance.go` which would improve readability